libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -114,12 +119,15 @@ static DNNReturnType get_input_ov(void *model, DNNData *input, const char *input
             input->width    = dims.dims[3];
             input->dt       = precision_to_datatype(precision);
             return DNN_SUCCESS;
+        } else {
+            //incorrect input name
+            APPEND_STRING(all_input_names, model_input_name)
         }
 
         ie_network_name_free(&model_input_name);
     }
 
-    av_log(ctx, AV_LOG_ERROR, "Could not find \"%s\" in model\n", model_input_name);
+    av_log(ctx, AV_LOG_ERROR, "Could not find \"%s\" in model, all input(s) are: \"%s\"\n", input_name, all_input_names);
```

Comment:
did you try it?

Reply:
Yes, any expression inappropriate?

Reply:
where did you query all_input_names?

Reply:
in the 'else' of 'for loop' above, the 'for loop' compare every 'model_input_name' of model with the 'input_name', and if it mismatch, 'else' branch will fill current model_input_name into all_input_names